books/RayTracingTheNextWeek.html

```diff
@@ -663,17 +663,12 @@
         if (objects.empty()) return false;
 
         aabb temp_box;
-        bool first_true = objects[0]->bounding_box(t0, t1, temp_box);
-
-        if (!first_true)
-            return false;
-
-        output_box = temp_box;
+        bool first_box = true;
 
         for (const auto& object : objects) {
-            if (!objects[i]->bounding_box(t0, t1, temp_box))
-                return false;
-            output_box = surrounding_box(output_box, temp_box);
+            if (!object->bounding_box(t0, t1, temp_box)) return false;
```

Comment:
I can confirm that the line from source compiles.  Don't know when I'll get around to the others; want to make sure my vetting process is sound